src/framework/accessors/instance.ts

```diff
@@ -131,13 +132,13 @@ export const getType = (
 };
 
 export const getPatrolSlot = (
-  $: GameEngine,
+  $: GameState,
```

Comment:
Why have it take a state instead of an engine? I don't recall whether there's precedent for this one way or another, but we should make sure it's consistent (or else it could cause issues if this needs to use some other accessor as a dependency)

Reply:
I forget exactly. I probably changed it because it's easier to have a GameState than a GameEngine and I was calling this code from a place that had a GameState but not a GameEngine, while also being able to modify all places that already called this place to use a GameState (assuming the code was being called previously at all).